src/components/ConditionalFilter/FiltersArea.tsx

```diff
@@ -35,7 +36,7 @@ export const FiltersArea = ({ onConfirm }: FiltersAreaProps) => {
     }
 
     if (event.type === "leftOperator.onChange") {
-      updateLeftOperator(event.path, event.value);
+      updateLeftOperator(event.path, event.value as LeftOperand);
```

Comment:
**question**: are those types incompatible so you have use `as`? 🤔

Reply:
actually - yes, it has same fields except one: "type"